src/Store/StoreThread.re

```diff
@@ -209,9 +201,10 @@ let start =
     Features.updateSubscriptions(setup, newState, dispatch);
 
     onAfterDispatch(action);
-  };
+    runEffects();
+  }
 
-  let runEffects = () => {
+  and runEffects = () => {
```

Comment:
dispatching relied on `runEffects` being run on every tick, so when I removed the ticking I had to execute `runEffects` here instead. All tests pass, so I don't think anything relied on it being run continuously.